content/guides/tutorials/server/requirements/index.md

```diff
@@ -18,10 +18,12 @@ The following are required for DNN Platform 9.0 and DNN Evoq 9.0.
 *   Windows Server 2008 R2
```

Comment:
We may want to note that SP1 is required for Windows Server 2008 R2 (per https://docs.microsoft.com/en-us/dotnet/framework/get-started/system-requirements#supported-server-operating-systems)

Reply:
Updated